app/views/contacts/index.slim

```diff
@@ -0,0 +1,38 @@
+.container.our-team
+  .row
+    .col-sm-12.col-lg-12
+      h1
+        |  Our team
+      div.photos-container
+        = link_to image_tag("liakh.png", class: 'team-member-image'), "https://github.com/yashka713", target: "_blank"
+        = link_to image_tag("mazniy.jpg", class: 'team-member-image'), "https://github.com/Vitaliy-Maznyi", target: "_blank"
+        = link_to image_tag("kujda.jpg", class: 'team-member-image'), "https://github.com/PeterKujda", target: "_blank"
+.container
+  .row
+    .col-sm-6
+      .well
+        h3.info-label
+          |  Address:
+        p.info-text
+          | 16, Tarasovskaya st., Kiev
+        br
```

Comment:
please do not use `br` . It's better to use css `margin-top` or `margin-bottom`. So please correct styles and remove all the `br`